bench-tps/src/perf_utils.rs

```diff
@@ -12,6 +12,7 @@ use {
     },
 };
 
+#[allow(dead_code)]
```

Comment:
If it is only complaining about the `elapsed` field not being used, I would rather just remove the `elapsed` field.

I see that we set it here:
https://github.com/anza-xyz/agave/blob/6fb43389778b370fc434fac6e6fc6abd955af355/bench-tps/src/perf_utils.rs#L79-L82

But then doesn't look like we ever read `elapsed` in this function:
https://github.com/anza-xyz/agave/blob/6fb43389778b370fc434fac6e6fc6abd955af355/bench-tps/src/bench.rs#L1046-L1051

Reply:
fixed: [125bac2](https://github.com/anza-xyz/agave/pull/1346/commits/125bac23cdb2426fd416c408638c2e69cfd7dd84)